### ISSUE TITLE
Can you give us the systemd equivalent for the startup script? 

### ISSUE THREAD

**Issue by helenclarko** [Didstopia/rust-server#86]
**Is your feature request related to a problem? Please describe.**
Newer releases of Ubuntu do not have upstart available

**Describe the solution you'd like**
systemd alternative auto startup script

**Comment by Dids** [comment]
Not sure I follow. This Docker container image uses neither upstart nor systemd.

**Comment by helenclarko** [comment]
Thanks Dids,

I was referring to the help document created back in 2016 for Ubuntu 14.04.

I probably should have just used 14.04 which has upstart, but I want to check with you first incase you had something.

**Comment by Dids** [comment]
Do you happen to have a link to that? Still not entirely sure how this relates to this project/Docker image, as we have no control over any documentation outside of this repository. 🙂

**Comment by helenclarko** [comment]
Of course, this document here:
https://rust.didscraft.com/rust-server-on-linux-using-docker/#automaticstartup

I know it's not part of the repository, but I wasn't sure where else to ask for systemd alternative to upstart.

I really hope this was a page you created and I'm not asking in the wrong place.
Sorry if I am wasting your time.

**Comment by Dids** [comment]
Ahh thank you, I'd entirely forgotten about that guide. I'll make a note to update it!

**Comment by helenclarko** [comment]
No no, Thanks you @Dids

**Comment by Hardcore-fs** [comment]
why would you need to complicate   the docker with such a script.

by using the correct docker commands

`docker run   --restart unless-stopped`

docker would do the same..

**Comment by HalfEatenPie** [comment]
I've translated the upstart code to systemd code simply for my own usecase.  Feel free to modify as you need, although probably using `docker run --restart unless-stopped` is good enough for your own usecase.

Create a file in: `/etc/systemd/system/rust-server.service`

```

[Unit]
Description=Rust Server Container
Requires=docker.service
After=docker.service

[Service]
Restart=always
ExecStartPre=-/usr/bin/docker rm -f rust-server 2>/dev/null
ExecStartPre=-/usr/bin/docker pull didstopia/rust-server
ExecStart=/usr/bin/docker run -p 28015:28015 -p 28015:28015/udp -p 28016:28016 -p 8080:8080 --name rust-server --env-file /rust.env -v /rust:/steamcmd/rust --rm didstopia/rust-server
ExecStop=/usr/bin/docker stop rust-server

[Install]
WantedBy=default.target
```

Modify this code section as you will.  I don't offer any support for this.  It works well enough for my own usecase, but YMMV.

**Comment by Hardcore-fs** [comment]
you might have missed the (rust+) port?

**Comment by HalfEatenPie** [comment]
Good catch @Hardcore-fs.  I'm getting back into Rust and running a server again and I'm not familiar with this new Rust+.
